common/common.cpp

```diff
@@ -1844,14 +1855,14 @@ struct llama_model * llama_load_model_from_url(
         curl_easy_setopt(curl, CURLOPT_NOPROGRESS, 0L);
```

Comment:
Here progression of each split will be displayed in parallel, it can be improved later on, the actual result is good enough for a first version.